### PR TITLE
[.NET][WPF] Fix Action Fallback rendering for unknown elements

### PR DESCRIPTION
**Pull request by almedina-ms**
The issue was that actions with unknown type were not being called for rendering and any action whose fallback ended up not being a rendereable element was  not rendering either so we were trying to insert a null children (not valid)

Also, a scrollview for the warnings was added as the warning list was cropping the payload are and the rendered card area

Fixes https://github.com/Microsoft/AdaptiveCards/issues/2772

### CODE REVIEW

File: source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionSetRenderer.cs [hunk 1]
```diff
@@ -141,8 +145,8 @@ public static void AddActions(Grid uiContainer, IList<AdaptiveAction> actions, A
         private static List<AdaptiveAction> GetActionsToProcess(IList<AdaptiveAction> actions, int maxActions)
         {
             // only consider known actions for ActionsToProcess
-            var actionsToProcess = actions.Where(obj => obj.GetType() != typeof(AdaptiveUnknownAction)).ToList();
-            return actionsToProcess.Take(maxActions).ToList();
+            // var actionsToProcess = actions.ToList();
```
Comment (paulcam206) [inline source/dotnet/Library/AdaptiveCards.Rendering.Wpf/AdaptiveActionSetRenderer.cs]: >// var actionsToProcess = actions.ToList(); [](start = 12, length = 43)

can remove this :)

**Comment by shalinijoshi19** [comment]
@paulcam206/ @almedina-ms , is there a reason this is still open?

**Comment by paulcam206** [comment]
heh -- still not sure why the bot claims there aren't any pipelines associated...

**Pull request merged**